pkg/virt-operator/application.go

```diff
@@ -188,8 +191,11 @@ func (app *VirtOperatorApp) Run() {
 	defer close(stop)
 
 	app.informerFactory.Start(stop)
+	f := k8sinformers.NewSharedInformerFactoryWithOptions(app.clientSet, 0)
```

Comment:
Two things.

1. is it reasonable to limit this informer to CSR with kubevirt.io label? I know we're already introspecting the request to ensure it's us that made the request, i'd feel better if we never saw other requests though. 

2. can this get moved to be managed by informerFactory?

Reply:
To 1., if we use the k8s client libraries for it, we have no access to the CSRs which get created. But we really only sign such certificates where our service accounts have the right RBAC permissions.

To 2. Yes can be done.